values/qkm.yaml.gotmpl

```diff
@@ -48,16 +49,14 @@ ingress:
     kubernetes.io/ingress.class: nginx
     nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
     nginx.ingress.kubernetes.io/ssl-passthrough: "true"
-
+    nginx.ingress.kubernetes.io/auth-tls-pass-certificate-to-upstream: "true"
+    
   hosts:
     - host: {{ .Values.qkm.namespace }}.{{ .Values.domainName }}
       paths:
         - path: /
           pathType: Prefix
           servicePort: 8080
-        - path: /isalive
```

Comment:
isnt this needed anymore? do you mind to explain why?

Reply:
Here the TLS passthru no longer allows the path resolution.
So the liveness endpoint is resolved with another ingress